oioioi/statistics/views.py

```diff
@@ -116,3 +125,76 @@ def statistics_view(
             'links': links(request),
         },
     )
+
+
+@contest_admin_menu_registry.register_decorator(
+    _("Monitoring"),
+    lambda request: reverse(
+        'monitoring', kwargs={'contest_id': request.contest.id}
+    ),
+    condition=(is_contest_admin | is_contest_observer),
+    order=110,
+)
+@enforce_condition(
+    contest_exists & can_enter_contest & can_see_stats
+)
+def monitoring_view(request):
+    cur_time = UTC.localize(datetime.now())
```

Comment:
Perhaps it would be better to use `request.timestamp`? Currently this always assumes we're in UTC, which might not be always true

Reply:
Using `request.timestamp` is also important, because administrators have an option to change their time